### DISCUSSION TITLE
Ability to select multiple workspaces at the same time

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#757] [Ideas]
### What feature would you like to see?

Sometimes users may have to switch between tabs from different workspaces frequently.
Instead of having to repeatedly switch workspaces or breaking ones finely crafted grouping, it would be a great feature to be able to select multiple workspaces at once. Having both workspaces list their tabs in the tab/sidebar, but still being separated for example by a line and a small label showing the name of the workspace.

This could nicely integrate with #499, making the selected workspaces differentiable by colour as well.

A VERY simplistic example quickly made in GIMP:
![ZenMultiWorkspaceExample](https://github.com/user-attachments/assets/2d5c1bd2-6b05-4c28-819f-7561736251c4)

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/701_

**Comment by artsi0m** [comment]
> Sometimes users may have to switch between tabs from different workspaces frequently.
> Instead of having to repeatedly switch workspaces or breaking ones finely crafted grouping, it would be a great feature to be able to select multiple workspaces at once. Having both workspaces list their tabs in the tab/sidebar, but still being separated for example by a line and a small label showing the name of the workspace.

Agreed. I enjoyed this feature in OpenBSD's cwm window manager.
